### ISSUE TITLE
What meltwater forcing should we apply?

### ISSUE THREAD

**Issue by adele-morrison** [adele-morrison/panan-meltwater#1]
Insight from some recent papers:

**Dawson et al. 2025** applied meltwater anomalies from Golledge et al. 2019, based on just one ice sheet model.
RCP4.5 (+0.042 Sv) and RCP8.5 (+0.16 Sv) projections. These values include +30% above the Golledge projections to offset salinity restoring.
<img width="600" alt="Screenshot 2024-12-06 at 9 20 48 AM" src="https://github.com/user-attachments/assets/b118f44f-ec78-42ad-b7dd-377cfe43a097">
<img width="600" alt="Screenshot 2024-12-06 at 9 26 47 AM" src="https://github.com/user-attachments/assets/ff4f75a8-903a-4285-9666-8abf10c18ea1">

**Swart et al. 2023 (SOFIA)**
**Tier 1**: antwater (0.1 Sv fixed in time), spatially uniform along coastal gridcells
**Tier 2**: ISMIP6 projections, SSP126 (0.015 Sv fixed) and SSP585 (max 0.196 Sv, ramped over 2015-2100), spatially uniform along coastal gridcells.
**Tier 3**: antwater-ambe  (0.1 Sv fixed in time), spread only over Amundsen/Bellingshausen, 210-290 E.

**Comment by adele-morrison** [comment]
We decided it would be better to use a weaker perturbation (i.e. less than Hannah's RCP4.5 (+0.042 Sv)), because if we use too large a perturbation then dense water in both models will shut down too quickly and we won't be able to analyse differences between them.

Therefore we will test the SOFIA Tier 2 SSP126 (+0.015 Sv) perturbation in panan-01 and see if it is large enough to see a response over 10 years.

**Comment by adele-morrison** [comment]
We were also thinking to test a non-uniform spatial distribution (i.e. West Antarctic only). Just looking at the relevant SOFIA experiment and for the non-uniform distribution perturbation (antwater-ambe), they use 0.1 Sv. I'm thinking maybe this is less worthwhile to run this perturbation now, because we want to use less than 0.1 Sv, so then our perturbation would not match any of the SOFIA exps. What do you reckon @willaguiar?

**Comment by willaguiar** [comment]
> We were also thinking to test a non-uniform spatial distribution (i.e. West Antarctic only). Just looking at the relevant SOFIA experiment and for the non-uniform distribution perturbation (antwater-ambe), they use 0.1 Sv. I'm thinking maybe this is less worthwhile to run this perturbation now, because we want to use less than 0.1 Sv, so then our perturbation would not match any of the SOFIA exps. What do you reckon @willaguiar?

Makes sense. I was worried about the uniform distribution before just because I thought it would look like the 60S distribution on the SOFIA experiments, But the SOFIA Tier 2 SSP126 coastal one sounds good to me.

**Comment by willaguiar** [comment]
I was thinking a little more about the coastal distribution... The surface area of cells in panan005 are smaller than the panan01, which could make the coastal freshwater fluxes in the individual cells higher in the panan005.... would that be an issue? i.e., do we want to fix the coastal distribution between then panan01 and panan005?

**Comment by adele-morrison** [comment]
I don't think the different horizontal spread due to resolution will make much of a difference. It will probably get spread out laterally pretty fast due to coastal current/eddies etc. They do discuss this in the SOFIA paper, perhaps we should have a read of that.

**Comment by adele-morrison** [comment]
I created a meltwater forcing file for the SOFIA Tier 2 SSP126 (+0.015 Sv) perturbation here:
`/g/data/g40/akm157/model_data/mom6/panan/RYF.runoff_all_SOFIA_ssp126.1990_1991.nc`

And the notebook to create it is [here](https://github.com/adele-morrison/panan-meltwater/blob/main/notebooks/forcing/forcing_create_meltwater_anom.ipynb).

**Comment by adele-morrison** [comment]
Another issue to think about is salinity restoring. The SOFIA protocol recommends saving salt restoring fluxes from the control and then reapplying these in the perturbation experiments, rather than letting the salt restoring adjust and overcompensate for the meltwater perturbation. I think since we're not exactly following the SOFIA protocol, it's probably ok to ignore this and just use salinity restoring - so long as we follow the same method in the 1/10th and 1/20th panan.

However, given that the salinity restoring is going to offset ~30% of the applied additional meltwater, perhaps we should do SOFIA ssp126 (0.015 Sv) + 30%, so total of 0.0195 Sv. What do you reckon?

**Comment by adele-morrison** [comment]
The ssp126 (0.015 Sv) looks like this so far:
<img width="600" alt="Screenshot 2024-12-14 at 11 33 20 AM" src="https://github.com/user-attachments/assets/cd71685b-8f04-4d04-88f0-71809208f094" />
<img width="600" alt="Screenshot 2024-12-14 at 11 33 34 AM" src="https://github.com/user-attachments/assets/6016589c-a09e-41d9-a4fc-da956715d96a" />

It definitely looks big enough that there will be a signal to investigate. But still maybe it would be more comparable to SOFIA ssp126 if we increase the meltwater forcing by 30%?

**Comment by willaguiar** [comment]
oh woow -  2 SV slowdown in just 10 years!!!

Personally, I think Id vote for doing the 30% offset of the salinity restoring to have the results more comparable to SOFIA (Im guessing, for a publication, the comparability can increase the impact of the results).

**Comment by willaguiar** [comment]
Complement: Do we expect the restoring fluxes (or offset %) to be different between the different resolutions? That might be something to take into account for the decision I guess.

**Comment by adele-morrison** [comment]
Well, possibly more like a 1.5 Sv slowdown, because the control is also drifting down. So ~25% reduction perhaps.

I don't think we expect the restoring to be different for different resolutions.

**Comment by adele-morrison** [comment]
Yep, that's right: ssp126-ismip-water, Tier 2.

My reading of Section A2.1 of Swart et al. 2023 is that the perturbations are only applied along the coast. There's a tier 3 experiment where they do south of 60S. I think the main difference between our experiment and SOFIA ssp126-ismip-water, Tier 2 is that we still have adaptive salinity restoring. I'll rerun now increasing the freshwater perturbation by 30% to offset the salinity restoring change.
